src/main/resources/org/jenkinsci/plugins/durabletask/PowershellScript/config.jelly

```diff
@@ -7,4 +7,7 @@
     <f:entry field="powershellBinary" title="Executable">
         <f:select />
     </f:entry>
+    <f:entry field="loadProfile" title="Load Profile">
```

Comment:
you could also help add text by creating a file called:
help-loadProfile.html in the same directory

only if you think it's needed though

Reply:
I think it's rather obvious, what this switch does. So I would go without a help text. But thanks for the pointer!